src/test/kotlin/com/igorwojda/integer/stepsgenerator/challenge.kt

```diff
@@ -4,7 +4,17 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 fun generateSteps(n: Int): List<String> {
-    TODO("not implemented")
+    var str = ""
+    val steps = mutableListOf<String>()
+    (1..n).forEach { row ->
+        (0 until n).forEach {
+            str += if (it < row) "#"
```

Comment:
please add proper formatting for if expressions, and don't rephrain from using paranthesis, things are much clearer with them, .e.g:
```
str += if (it < row) {
  "#"
} else {
 " "
}
```